lib/fastlane/plugin/appcenter/helper/appcenter_helper.rb

```diff
@@ -163,7 +165,7 @@ def self.update_symbol_upload(api_token, owner_name, app_name, symbol_upload_id,
       # upload symbol (dSYM or mapping) files to specified upload url
       # if succeed, then commits the upload
       # otherwise aborts
-      def self.upload_symbol(api_token, owner_name, app_name, symbol, symbol_type, symbol_upload_id, upload_url)
+      def self.upload_symbol(api_token, owner_name, app_name, symbol, symbol_type, symbol_upload_id, upload_url, timeout)
         connection = self.connection(upload_url, true)
 
         response = connection.put do |req|
```

Comment:
This is the place where the real file upload happens, as you can see from the file IO. This is where the timeout option should be added to the request.